src/components/booklist/booklist.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Book from '../book/book';
+import { connect } from 'react-redux';
+import './booklist.styles.scss'
+import { setLibrary } from '../../redux/actions/library';
+// import { API_URL } from '../../constants/constants';
+
+class BookList extends React.Component {
+
+	// componentWillMount() {
+		// fetch(`${API_URL}/books`, {
+	 //      method: 'GET',
+	 //      headers: {'AUTHORIZATION': `Bearer ${sessionStorage.jwt}`}
+	 //    })
+		// .then(res => res.json())
+		// .then(data => this.props.setLibrary(data))
+		// .catch(err => {console.log(err)});
+	// }
+
+	render () {
+		const { books } = this.props;
+		return (
+			<div className='booklist'>
+				{ books.map(({id, title, category, read}) => (
+					<Book key={id} id={id} title={title} category={category} read={read} />
+				))
+				}
+			</div>
+	)}
+}
+
+const mapStateToProps = state => ({
+ 	books: state.library.books.filter(book => state.filter.filter.length === 0 ? true : book.category === state.filter.filter),
```

Comment:
As mentioned in my comment on `library` reducer, we need to simplify the data structure we are using for the `library` state.